config/app.php

```diff
@@ -15,6 +15,14 @@
 
     'name' => env('APP_NAME', 'Urán'),
 
+    'version' => '3.0',
+
+    'logo_blue_path' => '/img/uran_blue.png',
```

Comment:
Do we need all of these here? This is a bit overcomplicated. You cold just create the config option 'logo' and set it's value to '/img/uran_blue.png'. If someone wants to change it, they will rewrite the value in the config.

Reply:
But what about the other 2? I use them too.

Reply:
Okay, then it's fine.